scalableminds/webknossos
# Call triggers/reload with fallback layer name

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open an annotation with a segmentation layer and brush anything
- In the "layers" tab on the left, select a color layer and hit "reload data from server". you should see how the buckets are reloaded.
- Do the same for the segmentation layer you just brushed in. you should see how the buckets are reloaded. in the network tab, there should be no requests with status 400 (before there was an unsuccessful request to `/histograms`)
- In your local test instance, change the name on disk of the current segmentation layer.
- Reload the layer. The segmentation should now be black. Before, old cached data would have been reloaded. [see this conversation in the original issue](https://github.com/scalableminds/webknossos/issues/8087#issuecomment-2566277991)

### TODOs:
- [x] test for segmentation layers without fallback layers

### Issues:
- fixes #8087 
- fixes that `/histogram` route was called for non-color layers from "reload data from server" button

------
(Please delete unneeded items, merge only when none are left open)
